src/graphql-config/graphql-config.service.ts

```diff
@@ -18,12 +18,13 @@ export class GraphQLConfigService implements GqlOptionsFactory {
     const apolloSandbox = graphqlConfig.cors.apolloSandboxOrigin;
 
     return {
-      playground: graphqlConfig.playgroundEnabled,
+      context: ({ req, res }) => ({ req, res }),
+      playground: graphqlConfig.playground,
       debug: graphqlConfig.debug,
       mocks: false,
       autoSchemaFile: true,
       resolvers: { Money, IBAN },
-      plugins: [ApolloServerPluginLandingPageLocalDefault()],
+      // plugins: [ApolloServerPluginLandingPageLocalDefault()],
```

Comment:
is it commented by mistake?

Reply:
no, It should be researched about it, 
it's not possible to have plugins and playground together and because we need cookies in our app it should be a playground with setting property.
I'm not sure who define plugin with this value, 
I need more details of functionality under the hood, so just keep it here.

Reply:
@MohammadForoutan added that.

Reply:
it's not a good practice to keep commented code inside the git repository.

Reply:
It's better to be here, but u right, I try to find out and fix this

Reply:
we can have one playground
apollo playground or Default native graphql playground we can choose between on of them
`{ playground: false,
 plugins: [ ApolloServerPluginLandingPageLocalDefault() ] }`
 or just
 `{ playground: true }`